templates/courseware/course_about.html

```diff
@@ -185,25 +185,37 @@
         </header>
 
         <ol class="important-dates">
+	  % if course.enrollment_end:
+	    % if course.enrollment_end < timezone.now():
+             <li class="important-dates-item">
+	       <i class="icon icon-calendar"></i> <p class="important-dates-item-title">${_("Enrollment")}</p>
+	       <span class="important-dates-item-text start-date">${_("Closed")}</p>
+             </li>
+           % else:
+            <li class="important-dates-item">
+	      <i class="icon icon-calendar"></i><p class="important-dates-item-title">${_("End of Registration")}</p>
+	      <span class="important-dates-item-text start-date">${course.enrollment_end.strftime('%d %b %Y')}</span></li>
```

Comment:
Is this date formatting compatible with multiple languages? I suggest the following:

    <%! from django.utils.formats import date_format %>
    <span>${date_format(course.enrollment_end))</span>